shaman/tests/test_util.py

```diff
@@ -105,3 +106,54 @@ def test_pick_last_used_node(self, session, monkeypatch):
         n2.last_used = datetime.datetime.now()
         session.commit()
         assert n1 == util.get_next_node()
+
+
+class TestParseDistroRelease(object):
+
+    @py.test.mark.parametrize('version,codename', [('14.04', 'trusty'), ('16.04', 'xenial'), ('16.10', 'yakkety')])
+    def test_ubuntu_version_gets_both_parsed(self, version, codename):
+        c, v = util.parse_distro_release(version)
+        assert v == version
+        assert c == codename
+
+    @py.test.mark.parametrize('version,codename', [('14.04', 'trusty'), ('16.04', 'xenial'), ('16.10', 'yakkety')])
+    def test_ubuntu_codename_gets_both_parsed(self, version, codename):
```

Comment:
the first one passes the version, this one passes the codename. The results are the same